### PR TITLE
Control-c fix

### PR DESCRIPTION
**Pull request by lock9**
closes #475

### CODE REVIEW

File: neo-cli/Services/ConsoleServiceBase.cs [hunk 1]
```diff
@@ -336,6 +336,11 @@ private void RunConsole()
                 Console.ForegroundColor = ConsoleColor.Yellow;
                 string line = ReadLine()?.Trim();
                 if (line == null) break;
```
Comment (superboyiii) [inline neo-cli/Services/ConsoleServiceBase.cs]: Line 338 should be deleted.

Comment (shargon) [reply neo-cli/Services/ConsoleServiceBase.cs]: Yes @lock9

Comment (vncoelho) [reply neo-cli/Services/ConsoleServiceBase.cs]: deleted.

**Comment by lock9** [comment]
I did a mistake while committing the file.
@vncoelho  what OS are you using?

**Comment by vncoelho** [comment]
Debian based

**Comment by lock9** [comment]
@vncoelho  what did you do to get this exception? I'm not having any issues with exceptions in code, even when I force them in different places of the code

**Comment by vncoelho** [comment]
Check my last comment, @lock9.

**Comment by lock9** [comment]
I reviewed the branch and it does not impact this implementation, only my code because I have additional changes.

**Pull request closed without merging**